apps/test/unit/templates/teacherDashboard/AssignmentSelectorTest.js

```diff
@@ -164,260 +72,154 @@ describe('AssignmentSelector', () => {
     wrapper
       .find('select')
       .at(0)
-      .simulate('change', {target: {value: 'csd'}});
+      .simulate('change', {target: {value: 2}});
     assert.equal(wrapper.find('select').length, 2);
     const secondary = wrapper.find('select').at(1);
-    assert.equal(secondary.find('option').length, 2);
-    assert.equal(
-      secondary
-        .find('option')
-        .at(0)
-        .text(),
-      'Unit 1: Problem Solving'
-    );
-    assert.equal(
-      secondary
-        .find('option')
-        .at(1)
-        .text(),
+    assert.equal(secondary.find('option').length, 3);
+    assert.deepEqual(secondary.find('option').map(option => option.text()), [
+      'Unit 1',
+      'Unit 2',
       ''
-    );
+    ]);
     assert.deepEqual(wrapper.instance().getSelectedAssignment(), {
-      courseId: 29,
-      scriptId: null
+      courseOfferingId: 2,
+      courseVersionId: 4,
+      unitId: 5
     });
   });
 
-  it('can select script in second dropdown after selecting primary', () => {
+  it('can select unit in unit dropdown after selecting course offering', () => {
     const wrapper = shallow(<AssignmentSelector {...defaultProps} />);
     wrapper
       .find('select')
       .at(0)
-      .simulate('change', {target: {value: 'csd'}});
+      .simulate('change', {target: {value: 2}});
     wrapper
       .find('select')
       .at(1)
-      .simulate('change', {target: {value: 'null_168'}});
+      .simulate('change', {target: {value: 3}});
     assert.deepEqual(wrapper.instance().getSelectedAssignment(), {
-      courseId: 29,
-      scriptId: 168
+      courseOfferingId: 2,
+      courseVersionId: 4,
+      unitId: 3
     });
   });
 
-  it('hides second dropdown after selecting "" primary', () => {
+  it('hides unit dropdown after selecting blank course offering', () => {
     const wrapper = shallow(<AssignmentSelector {...defaultProps} />);
     wrapper
       .find('select')
       .at(0)
-      .simulate('change', {target: {value: 'csd'}});
+      .simulate('change', {target: {value: 2}});
     assert.equal(wrapper.find('select').length, 2);
     assert.deepEqual(wrapper.instance().getSelectedAssignment(), {
-      courseId: 29,
-      scriptId: null
+      courseOfferingId: 2,
+      courseVersionId: 4,
+      unitId: 5
     });
 
     wrapper
       .find('select')
       .at(0)
-      .simulate('change', {target: {value: ''}});
+      .simulate('change', {target: {value: '__noAssignment__'}});
     assert.equal(wrapper.find('select').length, 1);
     assert.deepEqual(wrapper.instance().getSelectedAssignment(), {
-      courseId: null,
-      scriptId: null
-    });
-  });
-
-  it('includes uncoursed script if section is assigned that script currently', () => {
-    // Eventually we don't expect this to be a plausible case, but until we do
-    // our migration, it's possible for a section to be assigned to a script that
-    // is in a course, and we want to make sure this still shows up in our
-    // dropdown.
```

Comment:
I think it would be worth either (1) confirming that we do not have production data in this state, or (2) testing that we still do something reasonable in this case (anything without a js error is ok with me).

Reply:
per offline convo, this is implemented / tested on the backend now, so its ok to remove the test here.